docs/integrations/sources/twiliotaskrouter.md

```diff
@@ -0,0 +1,57 @@
+# Twilio Taskrouter
+
+This page contains the setup guide and reference information for the Twilio Taskrouter source connector.
+
+## Prerequisites
+
+Twilio Taskroute HTTP requests to the REST API are protected with HTTP Basic authentication. You will use your Twilio Account SID as the username and your Auth Token as the password for HTTP Basic authentication. You will also have to use your Workspace ID as the sid if you wish to access a particular workspace or it's subresources.
+
+You can find your Account SID and Auth Token on your [dashboard](https://www.twilio.com/user/account).
+You can find the Workspace ID's from the taskrouter [box](https://console.twilio.com/develop/explore) in the list.
+
+See [docs](https://www.twilio.com/docs/taskrouter/api) for more details.
+
+## Setup guide
+## Step 1: Set up the Twilio Takrouter connector in Airbyte
+
+### For Airbyte Cloud:
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
+3. On the Set up the source page, enter the name for the Twilio connector and select **Twilio Taskrouter** from the <Source/Destination> type dropdown. 
+4. Enter your `account_sid`.
+5. Enter your `auth_token`.
+6. Enter your `sid`.
+7. Click **Set up source**.
+
+### For Airbyte OSS:
+
+1. Navigate to the Airbyte Open Source dashboard.
+2. Set the name for your source.
+3. Enter your `account_sid`.
```

Comment:
can you provide more twilio links to tell users where to find the account_sid, auth_token and sid? Users don't see sid in the UI, the label text is the description, which is `workspace Id`.